presto-main/src/main/java/com/facebook/presto/metadata/StaticFunctionNamespace.java

```diff
@@ -354,10 +355,12 @@
 import static java.util.Collections.singletonList;
 import static java.util.Objects.requireNonNull;
 import static java.util.concurrent.TimeUnit.HOURS;
+import static java.util.stream.Collectors.toMap;
 
 @ThreadSafe
 class StaticFunctionNamespace
 {
+    private static final Map<String, OperatorType> OPERATOR_TYPES = ImmutableMap.copyOf(Arrays.stream(OperatorType.values()).collect(toMap(OperatorSignatureUtils::mangleOperatorName, x -> x)));
```

Comment:
What about move mangle / unmangle operator into `OperatorType`?

Reply:
You mean move `OperatorSignatureUtils` to spi?

Reply:
I'm asking @martint and @dain why do we need to mangle operator names. Ideally I want to just remove this. But even if we have to do it, the "mangling" will be exposed through `FunctionMetadata.getName` anyways I don't see why we cannot just initialize all operators with like `SUM("+", "$operator$SUM")`